src/App.js

```diff
@@ -23,15 +24,18 @@ const App = () => {
         </HelmetProvider>
         <Router history={history}>
           <Switch>
+            <PrivateRoute exact path={routes.lobby()}>
+              <LobbyPage />
+            </PrivateRoute>
             <NotLoggedInRoute exact path={routes.login()}>
               <LoginPage />
             </NotLoggedInRoute>
+            <NotLoggedInRoute exact path={routes.login_guest()}>
```

Comment:
Can we make this a default view instead of email login?

Reply:
👍

Reply:
fixed